Rakefile

```diff
@@ -10,7 +10,7 @@ if windows
   COPY = 'copy'
   DEL = 'del'
 else
-  MAKE = 'make'
+  MAKE = 'make -j'

```

Comment:
If you want parallel building, pick it up from an environment variable. The default should be no parallel.